src/Razor/test/Microsoft.AspNetCore.Razor.LanguageServer.Test/Formatting_NetFx/HtmlFormattingCollection.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT license. See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Razor.Formatting;
+using Xunit;
+
+namespace Microsoft.AspNetCore.Razor.LanguageServer.Formatting;
+
+[CollectionDefinition(Name)]
+public class HtmlFormattingCollection : ICollectionFixture<HtmlFormattingFixture>
```

Comment:
You are my hero! Thank you for adding a collection fixture to tame this test infrastructure!